feature-utils/poly-look/src/css/themes.css

```diff
@@ -21,15 +21,37 @@
 .poly-theme-light {
   --theme-primary-color: var(--color-gray-50);
   --theme-secondary-color: var(--color-indigo-950);
+  --theme-tertiary-color: var(--color-red-300);
   --theme-small-print-color: var(--color-gray-500);
-  --theme-info-button-color: var(--color-gray-300);
 }
 
 .poly-theme-dark {
   --theme-primary-color: var(--color-indigo-950);
   --theme-secondary-color: var(--color-gray-50);
+  --theme-tertiary-color: var(--color-red-300);
   --theme-small-print-color: var(--color-gray-400);
+}
+
+.poly-theme-light, .poly-theme-dark {
+  --pl-button-color-fill-primary: var(--theme-secondary-color);
+  --pl-button-color-text-primary: var(--theme-primary-color);
+  --theme-info-button-color: var(--color-gray-300);
+  --pl-button-color-fill-secondary: var(--transparent);
+  --pl-button-color-border-secondary: var(--theme-secondary-color);
+  --pl-button-color-text-secondary: var(--theme-secondary-color);
+}
+
+.poly-theme-coop {
```

Comment:
I still have to finish looking through/thinking about this, but in the meantime, the "coop" theme should be moved from poly-look to the feature since it's feature specific. If at some point we decide to re-use it, we can just move it back.

Reply:
Tagging along here, I agree, and also, this way you only need to change the values of 
- --theme-primary-color
- --theme-secondary-color
- --theme-tertiary-color
since the button is already taking in those variables

That would be the power of using alias tokens  and global tokens.

Reply:
As I commented previously, I still don't understand very well if we're having two features or just one with this new theme. However, independent of that, the `Screen` component that uses `--theme-primary-color` as background-color and the `PolyButton` are in poly-look.